Document/0x05b-Basic-Security_Testing.md

```diff
@@ -66,6 +66,58 @@ Once you've configured the network and established a connection between the test
 
 After completing these steps and starting the app, the requests should show up in the interception proxy.
 
+As mentioned before starting at Android 7, android device won't trust user CA certificates. In the following we will be presenting two methods to bypass this android security mesure.
+
+#### Network Security Configuration
+
+Available since Android 7,this feature allows apps to customize their network security settings,by precising which CA certificates the app will be trusting.This CA certificates can be system trusted CAs as well as user non-public CAs or both.
+Network security configuration uses an XML file where the app specify which CAs certificates will be trusting.
+
+res/xml/network_security_config.xml:
+```shell
+<?xml version="1.0" encoding="utf-8"?>
+<network-security-config>
+    <base-config>
+        <trust-anchors>
+            <certificates src="@raw/extracas"/>
+            <certificates src="system"/>
+        </trust-anchors>
+    </base-config>
+</network-security-config>
+```
+The apps must include an entry in the manifest file to point to the network security configuration file.
+```shell
+<?xml version="1.0" encoding="utf-8"?>
+<manifest ... >
+    <application android:networkSecurityConfig="@xml/network_security_config"
+                    ... >
+        ...
+    </application>decomp
+</manifest>
+```
+
+In order to intercept an application traffic which Android version is 7 and above, you must follow the steps below:
+
+- Decompile the app using decompiling tools.[Manual static Analysis] provides details about decompiling (https://github.com/OWASP/owasp-mstg/blob/master/Document/0x05b-Basic-Security_Testing.md#manual-static-analysis)
+- Make the application trust the proxy's certificate by creating a network security configuration with the giving certificate as explained above
+- Repackage the app.[Android developer documentation] explains how it's done. (https://developer.android.com/studio/publish/app-signing#signing-manually)
```

Comment:
Similar to comment above: add the url directly after the brackets (e.g.: 
[Android developer documentation](https://developer.android.com/studio/publish/app-signing#signing-manually)